app/models/favorites.php

```diff
@@ -0,0 +1,63 @@
+<?php
+class Favorites extends AppModel
+{
+    public static function vote($user_id, $comment_id, $action)
+    {
+        $db = DB::conn();
+        try {
+            $db->begin();
+            if ($action === 'favorite') {
```

Comment:
Please avoid magic strings :bow: Also, instead of string, I think you should pass a boolean value instead if it's for adding Favorite or not since the value is for only 2 possibilities, correct?